### DISCUSSION TITLE
My Account is Blocked

### DISCUSSION THREAD

**Discussion by rioprnta** [vercel/community#3789] [Account Issue] [answered]
### Question

I got message "This user account is blocked." when login with Github.

**Comment by amyegan** [comment, marked as answer]
Due to the sensitive nature of account information, we cannot safely discuss it in this public forum. For more info, please check this guide: [Why has my account or deployment been paused?](https://vercel.com/guides/why-is-my-account-deployment-blocked)

**Reply by rioprnta** [reply]
I still can't login with github until now.

**Reply by rioprnta** [reply]
Why has this issue been closed?
